server/src/middlewares/webAppProxyMiddleware.ts

```diff
@@ -30,6 +30,11 @@ const addCanonicalLinkToHTML = (html: string, href: string): string => {
   )
 }
 
+const addNoIndexToHTML = (html: string): string =>
+  html.replace('<head>', `<head><meta name="robots" content="noindex" />`)
+
+const searchResultsPageRegex = /\/recherche\?.*view=("|%22)Results("|%22)/
```

Comment:
@bruno-ebstein-pass-culture Est-ce que les urls qu'on reçoit côté proxy sont toujours encodées ? Histoire de simplifier à ça :
```suggestion
const searchResultsPageRegex = /\/recherche\?.*view=%22Results%22/
```